bin.js

```diff
@@ -52,52 +52,36 @@ const result = commist()
         'upload-url'
       ],
       boolean: [
-        'help'
+        'help',
+        'private'
       ],
       default: {
-        'upload-url': 'https://upload.clinicjs.org'
+        'upload-url': DEFAULT_UPLOAD_URL
       }
     })
 
     if (args.help) {
       printHelp('clinic-upload')
     } else if (args._.length > 0) {
-      checkMetricsPermission(() => {
+      checkMetricsPermission(async () => {
```

Comment:
Can you add a comment explaining that `checkMetricsPermission` does not add a catch handler, and as such we are wrapping all the content of this function in a try/catch? Can you also move the `trackEvent` call inside the try-catch?